jetty-core/jetty-jmx/src/main/java/org/eclipse/jetty/jmx/ConnectorServer.java

```diff
@@ -58,6 +59,7 @@ public class ConnectorServer extends AbstractLifeCycle
 {
     public static final String RMI_REGISTRY_CLIENT_SOCKET_FACTORY_ATTRIBUTE = "com.sun.jndi.rmi.factory.socket";
     private static final Logger LOG = LoggerFactory.getLogger(ConnectorServer.class);
+    public static final int DEFAULT_PORT = 1099;
```

Comment:
Make it `private` and rename to `DEFAULT_REGISTRY_PORT`.

Reply:
Done.